bin/api2html.js

```diff
@@ -19,7 +19,8 @@ const languageMap = {
     "ruby": "Ruby",
     "python": "Python",
     "java": "Java",
-    "go": "Go"
+    "go": "Go",
+    "php: "PHP"
```

Comment:
There's a missing `"` after `"php` which causes build failures with 0.3.1

```
/node_modules/api2html/bin/api2html.js:23
00:00:57.282      "php: "PHP"
00:00:57.282             ^^^
00:00:57.282  
00:00:57.282  SyntaxError: Unexpected identifier
00:00:57.282      at new Script (vm.js:74:7)
00:00:57.282      at createScript (vm.js:246:10)
00:00:57.282      at Object.runInThisContext (vm.js:298:10)
00:00:57.282      at Module._compile (internal/modules/cjs/loader.js:670:28)
00:00:57.282      at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
00:00:57.282      at Module.load (internal/modules/cjs/loader.js:612:32)
00:00:57.282      at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
00:00:57.282      at Function.Module._load (internal/modules/cjs/loader.js:543:3)
00:00:57.282      at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
00:00:57.282      at startup (internal/bootstrap/node.js:238:19)
```

Reply:
This should be resolved when PR #20 is merged